### PR TITLE
Add CytoCAD

### PR DESCRIPTION
**Pull request by cytham**
CytoCAD is a bioinformatics tool for the identification of large genomic copy-number variation through coverage anomaly detection (CAD) using mapped whole-genome sequencing (WGS) data. 

Basic information:
* Takes as input a mapped whole-genome sequencing BAM file and output a BED file and a chromosome ideogram figure
* Uses Ruptures python package for change point detection of read coverage data per chromosome
* Uses tagore for chromosome ideogram illustrations

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by cytham** [comment]
@BiocondaBot please update

**Comment by cytham** [comment]
@BiocondaBot please add label

File: recipes/cytocad/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+test:
+  imports:
+    - cytocad
+  commands:
+    - cytocad --help
+
+about:
+  home: https://github.com/cytham/cytocad
+  license: "GNU General Public v3 (GPLv3)"
```
Comment (bgruening) [inline recipes/cytocad/meta.yaml:43]: Can you please use https://spdx.org/licenses/ identifiers.

File: recipes/cytocad/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 22289cdca9ebf5627dda3acc74f869c4cd73421eed41eb6680505f6e077d6e0a
+
+build:
+  skip: True # [py<36 or osx]
```
Comment (bgruening) [inline recipes/cytocad/meta.yaml:13]: remove the python skip here and add this pinning to the host section please

**Review by bgruening — COMMENTED** [review]
Thanks looks great, 2 small comments.

**Review by bgruening — APPROVED** [review]

File: recipes/cytocad/meta.yaml (hunk 1 above)
Comment (cytham) [reply recipes/cytocad/meta.yaml]: Ok, thanks!

File: recipes/cytocad/meta.yaml (hunk 2 above)
Comment (cytham) [reply recipes/cytocad/meta.yaml]: Ok, thanks!

**Pull request merged**